src/adaPets.js

```diff
@@ -1,43 +1,81 @@
 // Use Node-style imports for dependencies.
-const axios = require('axios');
-const result = require('./result.js');
+const axios = require("axios");
+const result = require("./result.js");
 
 const setResult = result.setResult;
 const setError = result.setError;
 
-const BASE_URL = 'http://localhost:3000/pets/';
+const BASE_URL = "http://localhost:3000/pets/";
 
 // Option functions.
 const listPets = () => {
-  // Fill out as part of Wave 1.
+    // Fill out as part of Wave 1.
+    axios
+        .get(BASE_URL)
+        .then((response) => {
+            setResult(response.data);
+        })
+
+    .catch((error) => {
+        setError(`${error.message} in list pets`);
+    });
```

Comment:
looks like your indentation got away from you. Here's how we'd prefer indentation at Ada:
```suggestion
    axios.get(BASE_URL)
    .then((response) => {
        setResult(response.data);
    })
    .catch((error) => {
        setError(`${error.message} in list pets`);
    });
```